packages/headless/src/controllers/core/interactive-result/headless-core-interactive-result.ts

```diff
@@ -82,7 +83,7 @@ export function buildInteractiveResultCore(
   let longPressTimer: NodeJS.Timeout;
 
   return {
-    select: action,
+    select: debounce(action, defaultDelay, {isImmediate: true}),
```

Comment:
Shouldn't we use `options.selectionDelay` instead of `defaultDelay` here, so a user is able to override the delay?